webpages/konOpas_func.php

```diff
@@ -157,12 +158,17 @@ function retrieveKonOpasData($showpubstatus = 2, $showbio = 1) {
         if (!empty($row["htmlbio"])) {
             $bio = $row["htmlbio"];
         }
+        $links = [];
+        if (!empty($row['approvedphotofilename'])) {
+            $links['img'] = 'http' . ($_SERVER['HTTPS'] ? 's' : '') . '://' . $_SERVER['SERVER_NAME'] . PHOTO_PUBLIC_DIRECTORY . '/' . $row['approvedphotofilename'];
```

Comment:
This is a good point.
The link type was inherited from KonOpas. Accepted options are "img", "photo", "img_256_url" (the last one from Grenadine, apparently).
But rather than force a fixed link type, it makes sense to put the link type in a configuration setting.